tests/test_cmd_signature.py

```diff
@@ -882,6 +882,59 @@ def test_sig_describe_1_hp(c):
         assert cleaned_line in expected_output
 
 
+def test_describe_bug_905():
+    # Explicitly test https://github.com/dib-lab/sourmash/issues/905
+    with utils.TempDirectory() as location:
+        testdata1 = utils.get_test_data('protein_905.sig')
+
+        status, out, err = utils.runscript('sourmash',
+                                           ['signature', 'describe',
+                                            testdata1],
+                                           in_directory=location)
+        expected_output == """== This is sourmash version 3.2.2. ==
+== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==
+
+loaded 3 signatures total.
+---
+signature filename: test.sig
+signature: test.prot
+source file: test.prot
+md5: 57ae47c24a08a24b630dae80f0c6e256
+k=11 molecule=protein num=0 scaled=20 seed=42 track_abundance=1
```

Comment:
I think we can check for this line (and the equivalent on the other two sigs) instead of all the lines in the expected output.

This test as is will always require very stringent formatting of the `sig describe` output, which we might change in the future.